js_modules/dagit/packages/core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -238,30 +241,29 @@ const AssetSearch = ({assets}: {assets: Asset[]}) => {
   return (
     <div style={{width: 600}}>
       <Popover
-        minimal
-        fill={true}
         isOpen={open && matching.length > 0}
-        position={'bottom-left'}
+        position="bottom-left"
+        fill
         content={
-          <Menu style={{maxWidth: 600, minWidth: 600}}>
-            {matching.slice(0, 10).map((asset, idx) => (
-              <MenuItem
+          <MenuWIP style={{maxWidth: 600, minWidth: 600}}>
+            {matching.map((asset, idx) => (
```

Comment:
I'm not sure if this slice(0,10) was added for performance reasons but it seems like a subtle change here

Reply:
Ah, I just moved the slice up to where `matching` is defined.